arangod/Aql/tokens.ll

```diff
@@ -214,7 +214,7 @@ class Parser;
   return T_NONE;
 }
 
-(?i:AT[ \t\r\n]+LEAST) {
+(?i:AT[ \t\r\n]+LEAST)/[ \t\r\n] {
```

Comment:
I can't think of an AQL construct that would allow you to write e.g. `at leastblarg` and be valid somehow, so this one is probably not necessary, but maybe still good for futureproofing.

Reply:
Removed because it doesn't fix anything concrete